test/jdk/java/util/TimeZone/runCustomTzIDCheckDST.sh

```diff
@@ -0,0 +1,56 @@
+#
```

Comment:
I'd change this script into a java test case (using `.sh` is not recommended). In fact, this causes a failure invoking `javac` with `-Xmx768m` (from TESTVMOPTS)
There are libraries under `jdk/test/lib/` for building test jvm process and tools.

Reply:
I did away with this .sh script and combined the shell script and java test into one. Initially the only purpose of this shell script was in order to set up the environment variable for the proceeding test. However, I learnt that all this could be done within java, and I consolidated both the files into one test now.